### ISSUE TITLE
Devmode fails with missing import

### ISSUE THREAD

**Issue by ak80** [tbroyer/gwt-maven-plugin#116]
I am switching to this plugin, on a GWT project that has used the old maven gwt plugin.

A *mvn package*, to build the artifact in production mode works fine. The application works as before.
But I am having trouble getting the devmode to work. I do a *mvn gwt:codeserver -pl client -am* were client is the module containing the GWT client.

I have an annotation (@GeneratePojoBuilder ) in some classes that give me the error: 
```
[INFO]          [ERROR] Errors in 'file:/C:/Users/projekte/myproject/core/src/main/java/de/projectXY/core/values/IpAddress.java'
[INFO]             [ERROR] Line 20: GeneratePojoBuilder cannot be resolved to a type
[INFO]             [ERROR] Line 6: The import net.karneim.pojobuilder.GeneratePojoBuilder cannot be resolved
```

I do not understand why this does only happend with the codeserver goal. The annotation is not important for GWT and should be ignored.

Any ideas where I can look?

**Comment by tbroyer** [comment]
Could it be that you're using `<scope>provided</scope>`?

See https://tbroyer.github.io/gwt-maven-plugin/codeserver.html
> Also, [with gwt:codeserver] dependencies by default use the runtime scope, whereas gwt:compile uses the compile scope; this allows some differences between gwt:compile and gwt:codeserver classpaths when needed, while working in identical ways most of the time

**Comment by ak80** [comment]
Thanks for the quick help! 

I found something else, I have based the project layout on the multi-module archetype and also incorporated the profiles. I just had to activate the dev profile *mvn gwt:codeserver -pl client -am -Denv=dev*

Sorry, for the mistake! still don't understand exactly what the issue was, but for today it is enough for me to get it one step further.

By the way, thank you for the plugin and the work you are putting into it. The archetype has been really helpful!
